make/modules/jdk.compiler/Gendata.gmk

```diff
@@ -23,6 +23,10 @@
 # questions.
 #
 
+default:all
+
+include $(SPEC)
+include MakeBase.gmk
```

Comment:
These shouldn't be needed. `ModuleWrapper.gmk` which is the file including this file already does all of this.